src/Core/Function.php

```diff
@@ -37,3 +39,12 @@ function abort(int $code, string $message)
     );
     die();
 }
+
+function db(){
+    static $database = null;
```

Comment:
Good use of `static` keyword so that you cache the database connection for better performance. This is a simple implementation for the Singleton pattern that we'll utilize when you get your hands on design patterns.